cars/v1/apm-tracing.ini

```diff
@@ -0,0 +1,10 @@
+[meta]
+description = APM Tracing
+type = mixin
+
+[config]
+base = vanilla,apm_tracing/base
```

Comment:
I'll try this, but things like x-pack also use vanilla as a base, and is someone managed to remove defaults, this will add it back. But I'm game to remove it. I'll give it a test and get back to you